hr_timesheet_overtime/models/hr_employee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import time
+from openerp import models, api, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    initial_overtime = fields.Float(string="Initial Overtime", default=0.0)
```

Comment:
for what is initial_overtime used for ?

Reply:
To transfer overtime hours from one period to another. Ex: an employee has 10 overtime hours in 2019. For 2020, HR Manager will change `initial_overtime` from 0 to 10 and `overtime_start_date` as well

Reply:
ok , maybe adding "year" somewhere in the field's label or adding help attribute in the field.

Reply:
I've added the help attr